apps/sasquatch/build.gradle

```diff
@@ -31,6 +31,14 @@ android {
             dimension "dependency"
             applicationIdSuffix ".jcenter"
         }
+        projectGooglePlay {
+            dimension "dependency"
+            applicationIdSuffix ".project"
+        }
+        jcenterGooglePlay {
+            dimension "dependency"
+            applicationIdSuffix ".jcenter"
+        }
         vanilla {
             dimension "pushLibrary"
```

Comment:
BTW @guperrot do we still need non firebase variant?

Reply:
We don't really use it nowadays.

Reply:
ok, we've filed a separate task to clean this up